packages/verification-vue-component/src/wasm/pdf.wasm.init.js

```diff
@@ -0,0 +1,46 @@
+// import pdfWasm from './pdf.wasm'
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import wasmExec from './wasm_exec.js'
+
+let pdfWasmModule = null
+let pdfWasmInstance = null
+
+export default {
+    async init () {
+        console.log("WASM LOADED")
+        // Execute raw js code (wasm_exec.js)
+        // eslint-disable-next-line no-eval,no-useless-call
+        eval.call(null, wasmExec)
+
+        // eslint-disable-next-line no-undef
+        const go = new Go()
+
+        try {
+            // const wasm = cdfWasm({ });
+            // pdfWasmInstance = wasm
+
+            if (!WebAssembly.instantiateStreaming) { // polyfill
+                WebAssembly.instantiateStreaming = async (resp, importObject) => {
+                    const source = await (await resp).arrayBuffer()
+                    const instance = await WebAssembly.instantiate(source, importObject)
+                    return instance
+                }
+            }
+
+            const pdfWasmUrl = new URL('/wasm/pdf.a5957117.wasm', self.location.origin)
```

Comment:
Does it mean that we need to update this URL manually each time?

Reply:
Yes, that is the ugly bit i'll probably need to ask an FE dev. The issue is that this module does not know about that file because its in the certifaction-web project so i don't know how to link those properly.